src/store/components/AdminCollectionPage/style.less

```diff
@@ -1,3 +1,30 @@
 .admin-collection-page {
   margin-top: 1rem;
+
+  h2 {
+    font-size: 1.75rem;
+    font-weight: 500;
+    margin-bottom: 1rem;
+  }
+  &-form {
+    display: flex;
+    flex-direction: column;
+    max-width: fit-content;
+    &-field {
+      padding-bottom: 20px;
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      &-label {
+        font-size: 1.25rem;
+      }
+    }
+    &-buttons {
+      padding-bottom: 20px;
```

Comment:
Can we make these `padding-bottom: 1.25rem;` instead? I want to be consistent with units.